packages/dev/docs/src/Layout.js

```diff
@@ -70,11 +73,25 @@ function getTarget(href) {
   return '_blank';
 }
 
-function Page({children, title, styles, scripts}) {
+function dirToTitle(dir) {
+  return dir
+    .split('/')[0]
+    .split('-')
+    .map(w => w.charAt(0).toUpperCase() + w.slice(1))
+    .join(' ');
+}
+
+function Page({children, currentPage, publicUrl, styles, scripts}) {
+  let isSubpage = currentPage.name.split('/').length > 1 && !/index\.html$/.test(currentPage.name);
+  let pageSection = isSubpage ? dirToTitle(currentPage.name) : 'React Spectrum';
+  let keywords = currentPage.keywords.concat([currentPage.category, currentPage.title, pageSection]).join(',');
```

Comment:
I went to https://reactspectrum.blob.core.windows.net/reactspectrum/91802bdd857c937f55638c06cb1dc05ffd91c8ab/docs/index.html and found a leading comma and repeated keyword `,React Spectrum,React Spectrum`.

Reply:
👍 fixed